server/etcdserver/corrupt.go

```diff
@@ -47,30 +47,30 @@ func (s *EtcdServer) CheckInitialHashKV() error {
 		zap.String("local-member-id", s.MemberId().String()),
 		zap.Duration("timeout", s.Cfg.ReqTimeout()),
 	)
-
-	h, rev, crev, err := s.kv.HashByRev(0)
+	currentRev := s.kv.Rev()
```

Comment:
The same as above, it doesn't make much sense to introduce an unnecessary creation & close of a concurrentReadTx.

Reply:
This was intentional for code clarity, let's continue discussion in comment above.